configuration/messageproperties/messages_en.properties

```diff
@@ -850,6 +850,13 @@ pihcore.mch.between4and8=between 4 and 8
 pihcore.mch.greaterThan8=8+
 pihcore.mch.birthPlan=Birth Plan
 pihcore.mch.ultrasoundResultText=Ultrasound Results
+pihcore.mch.referral=Referral
+pihcore.mch.generalHistory=General History
+pihcore.mch.pregnancyHistory=Pregnancy History
+pihcore.mch.surgicalHistory=Surgical History
+pihcore.mch.menstrualHistory=Menstrual History
+pihcore.mch.gynecologyHistory=Gynecology History
+pihcore.mch.contraceptiveHistory=contraceptive History
```

Comment:
Is the "c" here supposed to be lowercase?

Reply:
oops, my mistake @mogoodrich.